go/src/koding/kites/kloud/provider/aws/aws.go

```diff
@@ -202,10 +202,10 @@ func (meta *Cred) Valid() error {
 		return errors.New("aws: secret key is empty")
 	}
 	if !reAccessKey.MatchString(meta.AccessKey) {
-		return errors.New("aws: access key is ill-formed")
+		return errors.New("aws: access key seems wrong")
```

Comment:
@ppknap Two things:

1) the implementation does not consider, neither for aws nor any other provider, that checks are fragile or can be invalidated - otherwise those would be only warnings and we would not be aborting credential verification if the check fail; moreover did any of the format actually changed till now?
2) wording for such error both for aws and google uses the word `ill-formed` - this issue is about using simpler language not about changing the meaning of it

I'll change `ill-formed` to `invalid` for all the provider after #10127 is merged.